### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-microej-sdk-6_0.13.0/

### CODE REVIEW

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 1]
```diff
@@ -3,6 +3,33 @@
...
+[0.13.0] - 2023-11-10
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+"""""
+
+- Add a check on EULA acceptation when using the MicroEJ Gradle plugin.
+- Automatically publish the ASSEMBLY_EXCEPTION.txt file if it exists at the root of the project.
+- Allow to publish the Feature file of an Application.
+- Add `:execTool` task to execute MicroEJ Tools provided by the selected VEE Port or Kernel.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:17]: Shouldn't we make it clear that only the stacktrace reader and the code coverage works (or at least are supported) for the moment? (cc @gjadi-mej )

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 2]
```diff
@@ -3,6 +3,33 @@
...
+
+Changed
+"""""""
+
+- Use VEE Port properties defined in the `LoadExecutableConfiguration` class to build a Feature.
+- Use the Kernel Executable as output of the `loadKernelExecutable` task instead of a directory.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:23]: These 2 items does not make sense for the end user IMO, they are about internal changes. Should we keep adding such changes in the changlelog? Not blocker though.

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 3]
```diff
@@ -3,6 +3,33 @@
...
+
+Fixed
+"""""
+
+- Fix warning during compilation because of non-existing file or folder.
+- Fix warning in SOAR when building an Executable with Architecture 8.0.0.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:29]: Can we add the text of the warning?

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 4]
```diff
@@ -3,6 +3,33 @@
...
+Changed
+"""""""
+
+- Use VEE Port properties defined in the `LoadExecutableConfiguration` class to build a Feature.
+- Use the Kernel Executable as output of the `loadKernelExecutable` task instead of a directory.
+
+Fixed
+"""""
+
+- Fix warning during compilation because of non-existing file or folder.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:28]: Can we add the text of the warning?

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 5]
```diff
@@ -3,6 +3,33 @@
...
+- Fix warning in SOAR when building an Executable with Architecture 8.0.0.
+- Enable Ant verbose mode for VEE Port scripts when Gradle debug log level is enabled.
+- Fix the build of a Feature when the provided Kernel does not contain the `dynamicFeatureLink.microejLaunch` build script.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:31]:
```suggestion
- Fix the build of a Feature when the provided Virtual Device does not contain the `dynamicFeatureLink.microejLaunch` build script (Virtual Device built with SDK 5).
```

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (gjadi-mej) [reply SDK6UserGuide/sdk6ChangeLog.rst:17]: IMHO having the documentation for only the tools we support is enough.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 5 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: Done.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 4 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: Done.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 3 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: Done.

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 2 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: I removed the 2 items from the changelog

File: SDK6UserGuide/sdk6ChangeLog.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: Ok, I modified  the changelog's entry.

File: SDK6UserGuide/sdk6ChangeLog.rst [hunk 6]
```diff
@@ -3,6 +3,33 @@
...
+Fixed
+"""""
+
+- Fix warning during compilation because of non-existing file or folder (``incorrect classpath: C:\\Users\\user\\...\myProject\\build\\resources\\main``).
+- Fix warning in SOAR when building an Executable with Architecture 8.0.0:
+  
+  .. code-block:: 
+  
+     1 : SOAR-L WARNING :
+         [M59] - Classpath file [C:\Users\user\...\myProject\build\resources\main] does not exist.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6ChangeLog.rst:28]: No need to display a code block IMO (note sure it is "allowed" in changelog). We can do like the line above: `Fix warning in SOAR when building an Executable with Architecture 8.0.0 ( [M59] - Classpath file [C:\Users\user\...\myProject\build\resources\main] does not exist. )`

Comment (denisbedin) [reply SDK6UserGuide/sdk6ChangeLog.rst]: Ok, done

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**